src/test/txvalidationcache_tests.cpp

```diff
@@ -17,15 +17,15 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
 
 BOOST_AUTO_TEST_SUITE(tx_validationcache_tests)
 
-static bool
-ToMemPool(const CMutableTransaction& tx)
-{
-    LOCK(cs_main);
+//static bool
+//ToMemPool(const CMutableTransaction& tx)
+//{
+//    LOCK(cs_main);
 
-    CValidationState state;
-    return AcceptToMemoryPool(mempool, state, MakeTransactionRef(tx), nullptr /* pfMissingInputs */,
-                              nullptr /* plTxnReplaced */, true /* bypass_limits */, 0 /* nAbsurdFee */);
-}
+//    CValidationState state;
+//    return AcceptToMemoryPool(mempool, state, MakeTransactionRef(tx), nullptr /* pfMissingInputs */,
+//                              nullptr /* plTxnReplaced */, true /* bypass_limits */, 0 /* nAbsurdFee */);
+//}
```

Comment:
can we remove commented code here

Reply:
If some day other comments are remove it'll not compile.

Reply:
Can you replace the deleted lines with:

@ todo: Re-enable tests ref commit acf994bed3b461e72b8c942b762465b5900f639a